tests/unit/models/util/ParseCalChart3File.spec.ts

```diff
@@ -0,0 +1,36 @@
+import Show from "@/models/Show";
+import LoadShow from "@/models/util/ParseCalChart3File";
+
+describe('models/util/ParseCalChart3File', () => {
+
+  describe('testLoad', () => {
+    let show: Show;
+
+    it('loading show', () => {
+      const show = LoadShow(new ArrayBuffer(8));
+      expect(show).toBeNull();
+    });
+    it('simple show', () => {
+      const base64_string = "SU5HTEdVMzVTSE9XAAABgFNJWkUAAAAEAAAAAUVORCBTSVpFTEFCTAAAAANBMABFTkQgTEFCTFNIRVQAAACGTkFNRQAAAAIxAEVORCBOQU1FRFVSQQAAAAQAAAAIRU5EIERVUkFQTlRTAAAACAf7AP1gAAAARU5EIFBOVFNDT05UAAAAEkVDTlQAAAACAABFTkQgRUNOVEVORCBDT05UUENOVAAAAAIAAEVORCBQQ05UQkFDSwAAAAQAAAAARU5EIEJBQ0tFTkQgU0hFVFNIRVQAAACLTkFNRQAAAAdzaGVldDIARU5EIE5BTUVEVVJBAAAABAAAAARFTkQgRFVSQVBOVFMAAAAIB/tA/aAAAQBFTkQgUE5UU0NPTlQAAAASRUNOVAAAAAIBAEVORCBFQ05URU5EIENPTlRQQ05UAAAAAgAARU5EIFBDTlRCQUNLAAAABAAAAABFTkQgQkFDS0VORCBTSEVUU0VMRQAAAAQAAAAARU5EIFNFTEVDVVJSAAAABAAAAAFFTkQgQ1VSUkVORCBTSE9X";
+      const data_array = Uint8Array.from(atob(base64_string), c => c.charCodeAt(0))
+      const show = LoadShow(data_array.buffer);
+      if (show) {
```

Comment:
Before this, we should run `expect(show).not.toBeNull();`
```suggestion
      expect(show).not.toBeNull();
      if (show !== null) {
```

Reply:
Can we also test `expect(show instanceof Show).toBeTruthy()`? Similar to what's happening in `Serializable.spec.ts`

Reply:
Done, I missed the not. part of the unit tester